src/dbnode/storage/index.go

```diff
@@ -400,6 +406,15 @@ func (i *nsIndex) SetRuntimeOptions(value runtime.Options) {
 	i.state.Unlock()
 }
 
+func (i *nsIndex) SetNamespaceRuntimeOptions(opts namespace.RuntimeOptions) {
+	// We don't like to log from every single index segment that has
+	// settings updated so we log the changes here.
+	i.logger.Info("set namespace runtime index options",
+		zap.Stringer("namespace", i.nsMetadata.ID()),
+		zap.Any("writeIndexingPerCPUConcurrency", opts.WriteIndexingPerCPUConcurrency()),
+		zap.Any("flushIndexingPerCPUConcurrency", opts.FlushIndexingPerCPUConcurrency()))
```

Comment:
nit; Should this be `WriteIndexingPerCPUConcurrencyOrDefault()`?

Reply:
Hm no, I wanted to know if its nil or not when we set it which is more information than just the default value if it's not set (can't tell if explicitly set or not if so).